src/script/src/python/coprocessor/compile.rs

```diff
@@ -122,11 +123,11 @@ pub fn compile_script(name: &str, deco_args: &DecoratorArgs, script: &str) -> Re
         );
     }
     // use `compile::Mode::BlockExpr` so it return the result of statement
-    python_compile::compile_top(
+    compile_top(
         &top,
         "<embedded>".to_owned(),
-        python_compile::Mode::BlockExpr,
-        python_compile::CompileOpts { optimize: 0 },
+        Mode::BlockExpr,
+        CompileOpts { optimize: 1 },
```

Comment:
does optimize = 1 not emit assert statements? if yes, maybe it's not suitable for our current project status? can it be set from outside like some command line opts?

Reply:
> does optimize = 1 not emit assert statements? if yes, maybe it's not suitable for our current project status? can it be set from outside like some command line opts?

Yes, only optimize=0 not emit assert statement(In python code), no, for now this three optimize is control by one `optimize: u8` option, but I think its suitable here, because we aren't writing `assert(...)` in python code anyway for now, so its' probably fine here(However just constant folding and dead code elimate is not a big deal, so I'm also open to change it back to `optimize: 0`